src/main/java/com/example/bookstoreapp/dto/BookUpdateDto.java

```diff
@@ -0,0 +1,20 @@
+package com.example.bookstoreapp.dto;
+
+import jakarta.validation.constraints.Min;
+import jakarta.validation.constraints.NotBlank;
+import jakarta.validation.constraints.Pattern;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Data
+public class BookUpdateDto {
+    @NotBlank
+    private String title;
+    @Min(value = 0)
+    private BigDecimal price;
+    private String author;
+    @Pattern(regexp = "^(978|979)[-]?\\d{10}$", message = "Invalid ISBN")
+    private String isbn;
+    private String description;
+    private String coverImage;
+}
```

Comment:
It's not clear for me , why we need that Dto ? It's the same as previous one

Reply:
It was misunderstanding in comments. I was suggested to create update dto while I was using BookRequestDto I understood that one is going to be for PATCH and another for PUT. As you can see one is anotated with @NotNull while another is not. At Q&A i discussed it and I let it stay in project.